x-pack/plugins/cases/server/telemetry/queries/cases.ts

```diff
@@ -52,17 +57,94 @@ export const getLatestCasesDates = async ({
   ]);
 
   return {
-    createdAt: savedObjects?.[0].saved_objects?.[0].attributes?.created_at ?? null,
-    updatedAt: savedObjects?.[1].saved_objects?.[0].attributes?.updated_at ?? null,
-    closedAt: savedObjects?.[2].saved_objects?.[0].attributes?.closed_at ?? null,
+    createdAt: savedObjects?.[0]?.saved_objects?.[0]?.attributes?.created_at ?? null,
+    updatedAt: savedObjects?.[1]?.saved_objects?.[0]?.attributes?.updated_at ?? null,
+    closedAt: savedObjects?.[2]?.saved_objects?.[0]?.attributes?.closed_at ?? null,
   };
 };
 
 export const getCasesTelemetryData = async ({
   savedObjectsClient,
   logger,
 }: CollectTelemetryDataParams): Promise<CasesTelemetry['cases']> => {
-  const byOwnerAggregationQuery = OWNERS.reduce(
+  try {
```

Comment:
Did some refactoring here, wrapped it all in a `try/catch` so we can more easily identify errors when we're developing additional telemetry and now doing a promise.all